server/src/services/players.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@nestjs/common';
+import { ResponseStatus } from '../interfaces/response.interface';
+import { Player } from '../interfaces/player.interface';
+
+@Injectable()
+export class PlayersService {
+  private readonly players: Map<string, Player> = new Map<
+    string,
+    Player
+  >();
+
+  constructor() {
+    // const player1: Player = {
+    //     id: "player1",
+    //     firstName: "Ala",
+    //     lastName: "Kowalska"
+    // }
+
+    // const player2: Player = {
+    //     id: "player2",
+    //     firstName: "Jan",
+    //     lastName: "Kowalski"
+    // }
+    // this.newPlayer(player1);
+    // this.newPlayer(player2);
```

Comment:
Przede wszystkim, nie warto zostawiać w "gotowych" commitach zakomentowanego kodu - później nie wiadomo czy ten kod jest ważny czy nie, można usunąć, a może lepiej zostawić? Do tego mamy git'a, aby pamiętać historię zmian kodu i nie musieć go "zostawiać" w komentarzach. Natomiast jeśli chodzi o ten konkretny komentarz tutaj: ponieważ w sprincie 1 tworzymy aplikację z zamockowanym spotkaniem, warto ten kod odkomentować. W kolejnych sprintach dodamy coś poważniejszego, na przykład bazę danych i odczytywanie zawodników, ale ponieważ stawiamy teraz na proste zakodowane na sztywno rozwiązanie, to musi być kilku zawodników dodanych z góry (tym bardziej, że nie mamy jeszcze panelu organizatora, który mógłby ręcznie dodać tych użytkowników, więc de facto z poziomu użytkownika nie ma możliwości dodania jakiegokolwiek zawodnika i muszą być oni wpisani na sztywno).